### PR TITLE
Make migration

### PR DESCRIPTION
**Pull request by Khalid-Abed**

### CODE REVIEW

File: database/migrations/2020_12_08_170104_create_education_table.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+class CreateEducationTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('education', function (Blueprint $table) {
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_08_170104_create_education_table.php]: The table name must be plural

File: database/migrations/2020_12_08_170104_create_education_table.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            $table->id();
+            $table->dateTime('start_date');
+            $table->dateTime('end_date');
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_08_170104_create_education_table.php]: We need only the date, no need for the date time

File: database/migrations/2020_12_08_180857_create_experiences_table.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            $table->unsignedBigInteger('industry_category_id');
+            $table->unsignedBigInteger('career_level_id');
+
+            $table->timestamps();
+            $table->foreign('user_id')->references('id')
+                ->on('users')->onDelete('cascade')
+                ->onUpdate('cascade');
+
+            $table->foreign('industry_category_id')->references('id')
+                ->on('industry_categories')->onDelete('cascade')
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_08_180857_create_experiences_table.php]: `onDelete("set null")` in stead of `cascade` and also needless to say make the column nullable

File: database/migrations/2020_12_08_180857_create_experiences_table.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                ->onUpdate('cascade');
+
+            $table->foreign('career_level_id')->references('id')
+                ->on('career_levels')->onDelete('cascade')
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_08_180857_create_experiences_table.php]: The same

File: database/migrations/2020_12_08_181645_create_languages_table.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('languages', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('name');
+            $table->text('proficiency');
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_08_181645_create_languages_table.php]: No need for `text` just use `string` and `64` as a second parameter

File: database/migrations/2020_12_09_112113_create_phone_numbers_table.php [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('phone_numbers', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->string('number');
```
Comment (Hossam-Tarek) [inline database/migrations/2020_12_09_112113_create_phone_numbers_table.php]: The `string` method default length is `255` and this is a huge space for just a phone number

File: app/Http/Controllers/KhaledAbedController.php [hunk 7]
```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\KhaledAbed;
+use Illuminate\Http\Request;
+
+class KhaledAbedController extends Controller
+{
```
Comment (Hossam-Tarek) [inline app/Http/Controllers/KhaledAbedController.php]: ??????????????

**Review by Hossam-Tarek — CHANGES_REQUESTED** [review]

**Review by Ahmed-Elhadary — APPROVED** [review]

**Review by Hossam-Tarek — APPROVED** [review]

**Pull request merged**